### PR TITLE
casio added to webdev (#208 issue resovled) as a part of GssoC'22

### PR DESCRIPTION
**Pull request by HarshitShukla01**
### Description

Added Casio to web development

### Issue
issue #208 resolved

### Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.

### CODE REVIEW

**Comment by prakhar3062** [comment]
@HarshitShukla01  add readme file for your project

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@HarshitShukla01 Add something about your project in `readme.md` file like @prakhar3062 said and yeah your project should under basic . Like this as below i mentioned 
webDevelopment/basic/casio.

**Comment by HarshitShukla01** [comment]
I am done the changes now pls review it.

**Comment by HarshitShukla01** [comment]
> [@Kushal997-das](/Kushal997-das) [Kushal997-das](/Kushal997-das) added the [ Pending ](/Kushal997-das/Project-Guidance/labels/Pending) label [43 minutes ago](#event-6182599398)

I am done the changes now pls review it.

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍 

@prakhar3062 Please check

File: Web Development/Basic/Casio/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- You can play songs using your keyboard or by clicking on keys displayed.
+- The main funcitoning of the project is done through JavaScript. 
+
+## Technology Used
+ - HTML
+ - CSS
+ - JavaScript 
+
+## Image
+![Cadio](demo.png "This is a sample image.")
```
Comment (prakhar3062) [inline Web Development/Basic/Casio/README.md:13]: Image is not visible. Please write the correct address

Comment (HarshitShukla01) [reply Web Development/Basic/Casio/README.md:13]: image

its visible for me

**Review by prakhar3062 — APPROVED** [review]
@photon149 okay from my side

**Review by photon149 — APPROVED** [review]
Looks good to me.
@Kushal997-das please merge the same in case of no conflicts.

**Pull request merged**